backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointResourceTest.java

```diff
@@ -1412,6 +1415,64 @@ void testConnectionCount() {
         }
     }
 
+    @Test
+    void testActive() {
+        String orgId = "queries-without-type";
+        String username = "user";
+        String identityHeaderValue = TestHelpers.encodeRHIdentityInfo(TestConstants.DEFAULT_ACCOUNT_ID, orgId, username);
+        Header identityHeader = TestHelpers.createRHIdentityHeader(identityHeaderValue);
+        MockServerConfig.addMockRbacAccess(identityHeaderValue, MockServerConfig.RbacAccess.FULL_ACCESS);
+
+        // stats[0] is the count
+        // stats[1] are the inactive ones
+        int[] stats = resourceHelpers.createTestEndpoints(TestConstants.DEFAULT_ACCOUNT_ID, orgId, 10);
```

Comment:
With `10`, the helper creates an even number of enabled and disabled endpoints, which doesn't help when assertions are run on `getMeta().getCount()` or `getData().size()`.
```suggestion
        int[] stats = resourceHelpers.createTestEndpoints(TestConstants.DEFAULT_ACCOUNT_ID, orgId, 11);
```

Reply:
oh - that's right - thanks!

Reply:
Updated to actually run the test correctly - Correctly failed when updated to use 11 endpoints.